.github/workflows/wheels.yml

```diff
@@ -193,7 +194,7 @@ jobs:
   # Build the source distribution under Linux
   build_sdist:
     name: Source distribution
-    runs-on: ubuntu-latest
+    runs-on: ubuntu-22.04
```

Comment:
What is the reason for this change?

Reply:
It is a change I've come across during my study, therefore I'm interested to know if other developers would agree if locking the `runs-on` version to a specific version. 

In terms of my reasoning why it would be better: If Github at any point in time moves to the next version of Ubuntu you will automatically also be moved over on your CI. If, in the unlikely case that there is a (security?) bug in this version of ubuntu you could experience negative side effects because of this.

Reply:
Security vulnerability are likely to be addressed in future bug fixes. In addition, this is likely that the vulnerabilities exist in previous version (if they are also LTS version then we should get support as well).

Basically, in the past we always have been required to manually bump the version and sometimes we don't really see the deprecation of the image. I think that I'm more convinced that using the latest in this case is less headache.

Reply:
ping @lesteve @ogrisel @thomasjpfan

Reply:
I think whether to pin the version of the image or not depends a bit on what the workflow is doing.

In this case I think the benefit that Guillaume described (automatic updates, less manual version bumping) are a good thing. This is because the work that happens in the workflow should reasonably "just work" on all reasonable Ubuntu base images.